doc/running.rst

```diff
@@ -46,6 +46,40 @@ environment variable, such as in the following example::
    $ srun -n 512 python -m mpi4py examples/wave-eager-mpi.py'
 
 
+OpenCL device selection
+-----------------------
+
+The :mod:`pyopencl` package supports selecting the device on which to run OpenCL code.
+There are multiple ways in which this selection can happen:
```

Comment:
Mention that it's always possible to write your own device selection code by using `pyopencl.get_platforms()`, `pyopencl.Platform.get_devices()`, and `pyopencl.Context`, so that this is merely a convenient shortcut.

Reply:
Added